### PR TITLE
Feature/mapbox getting started

### PR DESCRIPTION
**Pull request by JG1995**
Updated the Getting Started (Web v4) guide to also cover MapBox. Code examples are provided and approved by Esben. Also updated all JSFiddles to be on an actual MapsPeople account, instead of Simon's private account. There isn't a whole lot of new text compared to existing guide, but take a look.

### CODE REVIEW

File: src/web/v4/getting-started/js-fiddle-intro.md [hunk 1]
```diff
@@ -1 +1 @@
-If you have followed the tutorial exactly so far, you will now have something like this:
+If you have followed the tutorial exactly so far, you will now have something like this (due to technical limitations, only the Google Maps result is shown on this page, but MapBox implementation will also work if you followed this guide):
```
Comment (markhougaard) [inline src/web/v4/getting-started/js-fiddle-intro.md:1]:
```suggestion
If you have followed the tutorial exactly so far, you will now have something like this (due to technical limitations, only the Google Maps result is shown on this page, but Mapbox implementation will also work if you followed this guide):
```

File: src/web/v4/getting-started/map.md [hunk 2]
```diff
@@ -162,6 +165,145 @@ miMapElement.addEventListener('mapsIndoorsReady', () => {
 
 > For more information on how to configure the `<mi-map-googlemaps>` component, see [components.mapsindoors.com/map-googlemaps](https://components.mapsindoors.com/map-googlemaps/).
 
+</mi-tab-panel>
+<mi-tab-panel id="mb-manually">
+
+The MapsIndoors SDK is hosted on a Content Delivery Network (CDN) and should be loaded using a script tag.
+
+Insert the MapsIndoors SDK script tag into `<head>`, followed by the Mapbox script and style tag:
```
Comment (markhougaard) [inline src/web/v4/getting-started/map.md:173]:
```suggestion
Insert the MapsIndoors SDK script tag into `<head>`, followed by the Mapbox `script` and `style` tag:
```

File: src/web/v4/getting-started/map.md [hunk 3]
```diff
@@ -162,6 +165,145 @@ miMapElement.addEventListener('mapsIndoorsReady', () => {
 
 > For more information on how to configure the `<mi-map-googlemaps>` component, see [components.mapsindoors.com/map-googlemaps](https://components.mapsindoors.com/map-googlemaps/).
 
+</mi-tab-panel>
+<mi-tab-panel id="mb-manually">
+
+The MapsIndoors SDK is hosted on a Content Delivery Network (CDN) and should be loaded using a script tag.
```
Comment (markhougaard) [inline src/web/v4/getting-started/map.md:171]:
```suggestion
The MapsIndoors SDK is hosted on a Content Delivery Network (CDN) and should be loaded using a `script` tag.
```

File: src/web/v4/getting-started/search.md [hunk 4]
```diff
@@ -136,6 +139,126 @@ miMapElement.addEventListener('mapsIndoorsReady', () => {
 
 > For more information on available events and how to configure the `<mi-search>` component, see [components.mapsindoors.com/search](https://components.mapsindoors.com/search/).
 
+</mi-tab-panel>
+<mi-tab-panel id="mb-manually">
+
+MapsIndoors Locations can be retrieved in the mapsindoors namespace using the `LocationsService.getLocations()` method but first you need to add a `<input>` and `<button>` element to the DOM.
```
Comment (markhougaard) [inline src/web/v4/getting-started/search.md:145]: Do we need backticks on the `mapsindoors` word in the term "mapsindoors namespace"?

Comment (markhougaard) [inline src/web/v4/getting-started/search.md:145]:
```suggestion
MapsIndoors Locations can be retrieved in the mapsindoors namespace using the `LocationsService.getLocations()` method but first you need to add an `<input>` and `<button>` element to the DOM.
```

File: src/web/v4/getting-started/search.md [hunk 5]
```diff
@@ -136,6 +139,126 @@ miMapElement.addEventListener('mapsIndoorsReady', () => {
...
++ <input type="text" placeholder="Search">
++ <button onclick="onSearch()">Search</button>
+</body>
+</html>
+```
+
+* Create the `onSearch` method.
+* Get a reference to the search `<input>` element.
+* Define a new object with the search parameter `q` and the value of `searchInputElement`.
+* Call the `getLocations` method and log out the results to the console.
```
Comment (markhougaard) [inline src/web/v4/getting-started/search.md:176]: Styling question: Should we have periods at the end of list items? I think either way is fine, but we should try and be consistent 🙂 (I don't know which style we use the most—periods or no periods.)

File: src/web/v4/getting-started/search.md [hunk 6]
```diff
@@ -136,6 +139,126 @@ miMapElement.addEventListener('mapsIndoorsReady', () => {
...
++   });
++ }
+```
+
+> See all available search parameters in the [reference documentation](https://app.mapsindoors.com/mapsindoors/js/sdk/latest/docs/mapsindoors.services.LocationsService.html#.getLocations).
+
+</mi-tab-panel>
+<mi-tab-panel id="mb-components">
+
+Using the `<mi-search>` component you get a `<input>`element tied tightly together with the [Location Service](https://app.mapsindoors.com/mapsindoors/js/sdk/latest/docs/LocationsService.html).
```
Comment (markhougaard) [inline src/web/v4/getting-started/search.md:211]:
```suggestion
Using the `<mi-search>` component you get an `<input>`element tied tightly together with the [Location Service](https://app.mapsindoors.com/mapsindoors/js/sdk/latest/docs/LocationsService.html).
```

"tied tightly together" is a tongue-twister. Could it be rephrased?

File: src/web/v4/getting-started/search.md [hunk 7]
```diff
@@ -136,6 +139,126 @@ miMapElement.addEventListener('mapsIndoorsReady', () => {
...
+  </mi-map-mapbox>
+  <script src="main.js"></script>
++ <mi-search style="width: 600px;" mapsindoors="true" placeholder="Search">
++ </mi-search>
+</body>
+</html>
+```
+
+* Get a reference to the `<mi-search>` element.
+* Attach an `results` event listener and log out the results to the console.
```
Comment (markhougaard) [inline src/web/v4/getting-started/search.md:242]:
```suggestion
* Attach a `results` event listener and log the results in the console.
```

File: src/web/v4/getting-started/search.md [hunk 8]
```diff
@@ -440,6 +763,88 @@ miSearchElement.addEventListener('results', (event) => {
 + });
 ```
 
+</mi-tab-panel>
+<mi-tab-panel id="mb-manually">
+
+* Call `mapsIndoorsInstance.filter` with an array of location IDs.
```
Comment (markhougaard) [inline src/web/v4/getting-started/search.md:769]:
```suggestion
* Call `mapsIndoorsInstance.filter` with an array of Location IDs.
```

**Review by markhougaard — CHANGES_REQUESTED** [review]

File: src/web/v4/getting-started/search.md (hunk 5 above)
Comment (JG1995) [reply src/web/v4/getting-started/search.md:176]: For my personal writing style, I'd be inclined to be consistent WITH periods - But it's not something I've been super aware of previously, so I'll be consistent from here on out.

File: src/web/v4/getting-started/search.md (hunk 4 above)
Comment (JG1995) [reply src/web/v4/getting-started/search.md]: Probably a good idea.

File: src/web/v4/getting-started/search.md [hunk 9]
```diff
@@ -136,6 +139,126 @@ miMapElement.addEventListener('mapsIndoorsReady', () => {
 
 > For more information on available events and how to configure the `<mi-search>` component, see [components.mapsindoors.com/search](https://components.mapsindoors.com/search/).
 
+</mi-tab-panel>
+<mi-tab-panel id="mb-manually">
+
+MapsIndoors Locations can be retrieved in the mapsindoors namespace using the `LocationsService.getLocations()` method but first you need to add an `<input>` and `<button>` element to the DOM.
```
Comment (JG1995) [inline src/web/v4/getting-started/search.md:145]:
```suggestion
MapsIndoors Locations can be retrieved in the `mapsindoors` namespace using the `LocationsService.getLocations()` method but first you need to add an `<input>` and `<button>` element to the DOM.
```

**Review by markhougaard — APPROVED** [review]

**Pull request merged**